.github/workflows/link-validator.yml

```
@@ -0,0 +1,33 @@
+name: Link Validator
+
+on:
+  pull_request_target:
+    branches: [main, master]
+  schedule:
+    - cron:  '0 6 * * 1'
+
+jobs:
+  validate-links:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        # See https://github.com/actions/checkout/issues/299#issuecomment-677674415
+        ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Run Antora
+        run: make all
```

Is this supposed to be here? should it be

```suggestion
        run: sbt paradox
```

---

Thanks.